src/qt/rpcconsole.cpp

```diff
@@ -32,6 +32,7 @@
 #include <QKeyEvent>
 #include <QMenu>
 #include <QMessageBox>
+#include <QRegExp>
```

Comment:
I don't think this warrants adding `QRegExp` to the project

Reply:
We already have it. Maybe you are thinking of `QRegularExpression`?